packages/babel-core/src/config/partial.js

```diff
@@ -181,17 +195,23 @@ class PartialConfig {
   babelrc: string | void;
   babelignore: string | void;
   config: string | void;
+  isIgnored: boolean;
+  files: Array<string>;
```

Comment:
Should it be a `Set<string>`? There could be multiple `extends` in different config items:
```js
[{
  test: "foo.js",
  extends: "root-config"
}, {
  test: "bar.js",
  extends: "root-config"
}]
```